File: Qxbroker.API/Models/VerificationModel.cs
namespace Qxbroker.API.Models
{
    public class VerificationModel
    {
        public string Name { get; set; }

        public string Surname { get; set; }

        public string WalletAddress { get; set; }

        public string MobileNumber { get; set; }

        public IFormFile File { get; set; }
    }
}

File: qxbroker.client/src/app/models/verification.model.ts
import { BaseModel } from "./base.model";

export interface IVerification
{
    name: string;
	surname: string;
	mobileNumber: string;
	urlForDocumentImage: string;
	walletAddress: string;
	status: VerificationStatus;
}

export enum VerificationStatus {
	Unverified = 'Unverified',
	OnVerification = 'OnVerification',
	Rejected = 'Rejected',
	Verification = 'Verification',
}

export class VerificationModel extends BaseModel {
	name: string = '';
	surname: string = '';
	mobileNumber: string = '';
	urlForDocumentImage: string = '';
	walletAddress: string = '';
	status: VerificationStatus = VerificationStatus.Unverified;

	constructor(data: IVerification) {
		super();
		this.mapFromJson(data);
	}
}
